torch/_inductor/codegen/cpp.py

```diff
@@ -1458,7 +1458,11 @@ def load(self, name: str, index: sympy.Expr):
         else:
             line = f"at::vec::Vectorized<float>::loadu({loadbuf})"
         if non_contiguous:
-            tmpbuftype = "float" if is_mask else f"{DTYPE_TO_CPP[dtype]}"
+            tmpbuftype = (
+                "float"
+                if (is_mask and not opt_ctx.is_load_uint8_as_float)
```

Comment:
should you change how `is_mask` is initialized instead?

Reply:
Thanks for the comment. Change the initialized of `is_mask`.